cfy-go/nodes.go

```diff
@@ -127,18 +128,21 @@ func nodesOptions(args, options []string) int {
 			var deployment string
 			var nodeType string
 			var hostID string
+
 			operFlagSet.StringVar(&node, "node", "",
 				"The unique identifier for the node")
 			operFlagSet.StringVar(&deployment, "deployment", "",
 				"The unique identifier for the deployment")
-			operFlagSet.StringVar(&nodeType, "node-type",
-				cloudify.KubernetesNode, "Filter by node type")
```

Comment:
Use:
```
operFlagSet.StringVar(&nodeType, "node-type",
				os.Getenv("CFY_K8S_NODE_TYPE"), "Filter by node type")
```

Reply:
@0lvin check https://github.com/cloudify-incubator/cloudify-rest-go-client/tree/testing/cfy-go module it is up to date